lib/takeaway.rb

```diff
@@ -0,0 +1,24 @@
+class Takeaway
+  attr_reader :menu, :order, :total
+
+  def initialize
+    @menu = {
```

Comment:
In line with single responsibility principle, you could consider making the menu a separate class